.github/workflows/macos.yml

```diff
@@ -1,6 +1,10 @@
 name: macOS build
 
-on: [push, pull_request]
+on:
```

Comment:
Caviats:
- needs to be on default branch
- will then check that branch (PR activity unclear)
- `completed` means independent of status, additional `if: ${{ github.event.workflow_run.conclusion == 'success' }}` check needed in job